src/scripts.js

```diff
@@ -496,7 +527,11 @@ function getIngredientListElement(e, selectedRecipe) {
   ingredientListSection.innerHTML = '<h3>ingredients</h3>';
   const ingredientNames = selectedRecipe.determineRecipeIngredients(ingredientsData);
   const ingredientListText = selectedRecipe.ingredients.reduce((acc, ingredient, index) => {
-    acc += `${ingredientNames[index]}: ${ingredient.quantity.amount.toString()} ${ingredient.quantity.unit}<br><br>`;
+    let amount = ingredient.quantity.amount
+    if (!(ingredient.quantity.amount % 1) === 0) {
```

Comment:
We didn't like having infinitely long decimals, so added this in to clean it up.